srcpkgs/traefik/INSTALL

```diff
@@ -0,0 +1,4 @@
+case "${ACTION}" in
+post)
+	chown -R _traefik:_traefik /etc/traefik/traefik.yml ;;
```

Comment:
Should this really be owned/writable by daemon?

Reply:
It's definitly not necessary
Just thought it'd be sensible
root-owned works just as well